docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,324 @@
+        gender = int((int(gender)+1)/2)
+        return img, (torch.tensor(gender), torch.tensor([x_L, y_L, x_R, y_R], requires_grad=True),[w, h])
+
+    transform=tfms.Compose([
+        tfms.ToTensor(),
+        tfms.Normalize(imagenet_mean, imagenet_std)
+    ])
+
+    celeba_dataset = CelebADataset(data_root, transform)
+    celeba_dataloader = DataLoader(celeba_dataset, batch_size=batch_size, shuffle=True)
+    full_dataset_length = len(celeba_dataset)
+    indices = list(np.arange(full_dataset_length))
+    np.random.shuffle(indices)
+    train_indices = indices[math.floor(full_dataset_length * valid_split_percent):]
+    valid_indices = indices[:math.floor(full_dataset_length * valid_split_percent)]
+    train_dataset = Subset(celeba_dataset, train_indices)
+    valid_dataset = Subset(celeba_dataset, valid_indices)
+    train_dataloader = DataLoader(train_dataset, batch_size=batch_size, shuffle=True)
+    valid_dataloader = DataLoader(valid_dataset, batch_size=batch_size, shuffle=False)
+
+Here we can see how each dataset sample looks like:
+
+.. code-block:: python
+
+    print (train_dataset[0])
+
+Here, we can see an example from the training dataset. It shows an image of a person, printing the gender and also showing the location of the eyes.
+
+.. code-block:: python
+
+    sample_number = 16
+    image = train_dataset[sample_number][0]
+    image = image.permute(1,2,0).detach().numpy()
+    image_rgb = cv2.cvtColor(np.float32(image), cv2.COLOR_BGR2RGB)
+    Gender = 'male' if int(train_dataset[sample_number][1][0])==1 else 'female'
+    print('Gender is: ', Gender)
+    w, h = train_dataset[sample_number][1][2]
+    (x1, y1) = train_dataset[sample_number][1][1][0:2]
+    (x2, y2) = train_dataset[sample_number][1][1][2:4]
+    x1, x2 = int(x1*h), int(x2*h)
+    y1, y2 = int(y1*w), int(y2*w)
+    image_rgb	=	cv2.drawMarker(image_rgb, (x1,y1), (0,255,0))
+    image_rgb	=	cv2.drawMarker(image_rgb, (x2,y2), (0,255,0))
+    image_rgb = np.clip(image_rgb , 0, 1)
+    plt.imshow(image_rgb)
+    plt.axis('off')
+    plt.show()
+
+.. image:: /_static/img/classification_and_regression/dataset_sample.png
+
+Network
+=======
+
+Below, we define a new class, named 'ClassifierLocalizer, which accepts a pre-trained CNN and changes its last fully connected layer to be proper for the two task problem. The new fully connected layer contains 6 neurons, 2 for the classification task (male or female) and 4 for the localization task (x and y for the left and right eyes). Moreover, to put the location results on the same scale as the class scores, we apply the sigmoid function to the neurons assigned for the localization task.
+
+.. code-block:: python
+
+    class ClassifierLocalizer(nn.Module):
+        def __init__(self, model_name, num_classes=2):
+            super(ClassifierLocalizer, self).__init__()
+            self.num_classes = num_classes
+            
+            # create cnn model
+            model = models.__dict__[model_name](True)
```

Comment:
Prefer using `getattr` instead of `__dict__`.

Reply:
i'll change it

Reply:
done